content/post/105-deprecating-secio.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2020-08-06
+url: /2020-08-06-deprecating-secio/
+title: We're removing support for the SECIO security transport
+author: Jacob Heun
+tags: ipfs, libp2p, secio
+---
+
+We're removing support for the SECIO security transport in IPFS and libp2p. You can mitigate impact to your node(s) by updating to the latest versions. Newer nodes should not experience any signficant change when support is removed, but older nodes will start to experience performance degradation. Read on for more details.
+
+## What are security transports?
+
+Security transports are components of libp2p that provide end to end encryption of data that is sent across the network. Libp2p provides the ability to negotiate security transports once a connection is established, giving nodes the ability to support multiple security transports. Once a security transport has been negotiated, all data sent and received is encrypted and only the intended peer can decrypt it.
```

Comment:
> end to end encryption of data

This generally refers to client side encryption of data (e.g. encrypting data and adding it to IPFS) as opposed to transport layer encryption.

Perhaps something like this

```suggestion
Security transports are components of libp2p that encrypt data while it is being sent across the network. Libp2p provides the ability to negotiate security transports once a connection is established, giving nodes the ability to support multiple security transports. Once a security transport has been negotiated, all data sent and received is encrypted and only the intended peer can decrypt it.
```